SpatialGDK/Source/SpatialGDK/Private/Interop/CrossServerRPCSender.cpp

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/CrossServerRPCSender.h"
+
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Utils/SpatialMetrics.h"
+
+using namespace SpatialGDK;
+
+CrossServerRPCSender::CrossServerRPCSender(ViewCoordinator& InCoordinator, USpatialMetrics* InSpatialMetrics)
+	: Coordinator(InCoordinator)
+	, SpatialMetrics(InSpatialMetrics)
+{
+}
+
+void CrossServerRPCSender::SendCommand(const FUnrealObjectRef& InTargetObjectRef, UObject* TargetObject, UFunction* Function,
+									   RPCPayload&& InPayload, FRPCInfo Info, const TOptional<Trace_SpanId>& SpanId) const
+{
+	if (Function == nullptr || TargetObject == nullptr || InTargetObjectRef.Entity == SpatialConstants::INVALID_ENTITY_ID
+		|| Info.Type != ERPCType::CrossServer)
+	{
+		return;
+	}
+
+	const auto SchemaType = Schema_CreateCommandRequest();
+	Schema_Object* RequestObject = Schema_GetCommandRequestObject(SchemaType);
+	RPCPayload::WriteToSchemaObject(RequestObject, InTargetObjectRef.Offset, Info.Index, FMath::RandHelper(INT_MAX),
+									InPayload.PayloadData.GetData(), InPayload.PayloadData.Num());
+	CommandRequest CommandRequest(OwningCommandRequestPtr(SchemaType), SpatialConstants::SERVER_TO_SERVER_COMMAND_ENDPOINT_COMPONENT_ID,
```

Comment:
You can just create the CommandRequest object in the first place. The Owngin...Ptr types are just made as a migration thing. Rule of thumb is that you should never be calling Scheam_Create... outside of spatial view.